frontend/integration-tests/protractor.conf.ts

```diff
@@ -96,6 +96,7 @@ export const config: Config = {
     performance: ['tests/base.scenario.ts', 'tests/performance.scenario.ts'],
     serviceCatalog: ['tests/base.scenario.ts', 'tests/service-catalog/service-catalog.scenario.ts', 'tests/service-catalog/service-broker.scenario.ts', 'tests/service-catalog/service-class.scenario.ts', 'tests/service-catalog/service-binding.scenario.ts'],
     catalog: ['tests/base.scenario.ts', 'tests/catalog.scenario.ts'],
+    marketplace: ['tests/base.scenario.ts', 'tests/marketplace/kubernetes-marketplace.scenario.ts'],
     all: ['tests/base.scenario.ts',
```

Comment:
Should `marketplace` tests be added to the `all` group so that it is run during jenkins CI?

Reply:
I think so, I'll add it right now. Thanks!